nifi-asymmetrik-standard-bundle/nifi-asymmetrik-standard-reporting-tasks/src/main/java/com/asymmetrik/nifi/reporting/CloudwatchNiFiClusterMetricsReporter.java

```diff
@@ -131,10 +131,12 @@ public void startup(ConfigurationContext context) {
             if (property.isDynamic() && StringUtils.isNotBlank(value)) {
                 dynamicDimensions.add(new Dimension()
                         .withName(property.getDisplayName())
-                        .withValue(value)
+                        .withValue(context.getProperty(property)
+                                .evaluateAttributeExpressions()
```

Comment:
Can you also make this same change to InfluxNiFiClusterMetricsReporter? It looks like that one also advertises that dynamic properties support expression language but never evaluates them.

Reply:
Rick pushed the dynamic prop support for InfluxNiFiClusterMetricsReporter